codegen/src/main/java/software/amazon/awssdk/codegen/model/rules/endpoints/EndpointTestModel.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.codegen.model.rules.endpoints;
+
+import com.fasterxml.jackson.core.TreeNode;
+import java.util.Map;
+
+public class EndpointTestModel {
```

Comment:
Should we have `@SdkTestInternalApi` annotation for this  ?

Reply:
Sure

Reply:
Actually nevermind, this is in codegen. this model is not meant for tests, but to help generated tests...